BraintreeCore/src/main/java/com/braintreepayments/api/TLSCertificatePinning.kt

```diff
@@ -714,7 +712,7 @@ CkPa1mnI2g7vVq2d66WZmpsDtY1EwUya7vmZTxw7BFlntV7lrVxPLhUbq1wJGIbT
 uEnXgUGZxnLYZfCNsC/UAQxvKKpxBLWus/n1InM/97HfA31WJsvQ
 -----END CERTIFICATE-----"""
 
-    @get:Throws(SSLException::class)
-    val certInputStream: InputStream
-        get() = ByteArrayInputStream(CERTIFICATES.toByteArray(StandardCharsets.UTF_8))
+    private val certificateByteArray = CERTIFICATES.toByteArray(StandardCharsets.UTF_8)
+
+    fun createCertificateInputStream() = ByteArrayInputStream(certificateByteArray)
```

Comment:
Through debugging, I found that each networking client needs its own input stream for certificate pinning. When `BraintreeHttpClient` and `BraintreeGraphQLClient` share an input stream, an `SSLException` is thrown. I believe the first networking client to acquire the shared InputStream succeeds, but subsequent clients will fail because the singleton InputStream is closed.

Since a new `InputStream` is required to create each networking client, I decided to make its intent more explicit by refactoring the getter into a `createCertificateInputStream()` factory method.